package.json

```diff
@@ -157,7 +157,6 @@
         "windows": {
           "program": "./coreclr-debug/debugAdapters/OpenDebugAD7.exe"
         },
-        "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
```

Comment:
It does? The instructions for VS Code telemetry do not say to add this to the debuggers section. Or, is this used for the debug adapter telemetry (which I'm totally unfamiliar with).